.github/workflows/build.yml

```diff
@@ -160,18 +347,26 @@ jobs:
           name: wheels
           path: dist/*.whl
 
-  test-unix:
-    needs: build
+  test-windows:
+    needs:
+      - build-windows
+      - init
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        python-version: ["3.7", "3.8", "3.9"]
+        # os: ${{ fromJSON(needs.init.outputs.test_windows) }}
+        os: [windows-latest]
+        python-version: ${{ fromJSON(needs.init.outputs.python_versions) }}
         compiler: [gnu]
       fail-fast: true
+    defaults:
+      run:
+        shell: bash
```

Comment:
No idea why, but test failures occur only when running in bash, remove that stance to have the same behavior as before.  Remove it from all 3 test-* jobs, they were not set previously and are useless anyway on Unix.

Reply:
ok back to previous state.